tests/delete.js

```
@@ -0,0 +1,44 @@
+// delete endpoints
+var deleter = require('../src/delete');
+
+module.exports = function(expect) {
+    /* DELETE one of the projects endpoints and check whether it can still
+       be retrieved from the database */
+    describe('DELETE /projects', function() {
+        it('Deletes the desired project', function(done) {
+            deleter.deleteProject('Whats Fresh')
+            .then(function(project) {
+                expect(project).to.be.an('undefined');
+                //expect(res.statusCode).to.equal(404);
+                done();
+            });
+        });
+
+        it('Fails if it receives an invalid project', function(done) {
+            deleter.deleteProject('Nothing').then(function(project) {
+                //expect(res.statusCode).to.equal(400);
```

I think it should probably throw an error with the name of the project that was being deleted.